plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -82,42 +85,16 @@ class AutoQueueConf:
     # name of the job
     name: str = "${hydra.job.name}"
 
-    # following parameters are SLURM specific
-
-    # Maximum number of retries on job timeout.
-    # Change this only after you confirmed your code can handle re-submission
-    # by properly resuming from the latest stored checkpoint.
-    # check the following for more info on slurm_max_num_timeout
-    # https://github.com/facebookincubator/submitit/blob/master/docs/checkpointing.md
-    slurm_max_num_timeout: int = 0
-    # USR1 signal delay before timeout for the slurm queue
-    slurm_signal_delay_s: int = 30
-    # slurm partition to use on the cluster
-    slurm_partition: Optional[str] = None
-
-
-@dataclass
-class QueueParams:
-    slurm: SlurmQueueConf = SlurmQueueConf()
-    local: LocalQueueConf = LocalQueueConf()
-    auto: AutoQueueConf = AutoQueueConf()
-
-
-@dataclass
-class SubmititConf:
-    queue: QueueType = QueueType.slurm
-
-    folder: str = "${hydra.sweep.dir}/.${hydra.launcher.params.queue}"
-
-    queue_parameters: QueueParams = QueueParams()
+    # additional specific parameters and overrides are
+    # provided here
+    local_executor: BaseSubmititConf = BaseSubmititConf()
+    slurm_executor: SubmititSlurmConf = SubmititSlurmConf()
```

Comment:
Here are the configs for
- overriding the default/shared values above in a given executor
- adding cluster specific parameters

Since local executor does not have anything more, it just have a plain BaseSubmititConf for overriding the shared configuration